sim/druid/forms.go

```diff
@@ -263,144 +263,126 @@ func (druid *Druid) registerCatFormSpell() {
-// 			druid.PseudoStats.DamageTakenMultiplier /= potpdtm
-//			Switch to using AddStat as PseudoStat is being removed
-// 			druid.PseudoStats.BaseDodge -= 0.02 * float64(druid.Talents.FeralSwiftness+druid.Talents.NaturalReaction)
-
-// 			druid.AddStatsDynamic(sim, predBonus.Invert())
-// 			druid.AddStatsDynamic(sim, statBonus.Invert())
-// 			druid.RemoveDynamicEquipScaling(sim, stats.Armor, druid.BearArmorMultiplier())
-// 			if potpDep != nil {
-// 				druid.DisableDynamicStatDep(sim, potpDep)
-// 			}
-
-// 			healthFrac := druid.CurrentHealth() / druid.MaxHealth()
-// 			druid.DisableDynamicStatDep(sim, stamDep)
-// 			if hotwDep != nil {
-// 				druid.DisableDynamicStatDep(sim, hotwDep)
-// 			}
-// 			druid.RemoveHealth(sim, druid.CurrentHealth()-healthFrac*druid.MaxHealth())
-
-// 			if !druid.Env.MeasuringStats {
-// 				druid.AutoAttacks.SetReplaceMHSwing(nil)
-// 				druid.AutoAttacks.EnableAutoSwing(sim)
-
-// 				druid.manageCooldownsEnabled()
-// 				druid.UpdateManaRegenRates()
-// 				druid.EnrageAura.Deactivate(sim)
-// 				druid.MaulQueueAura.Deactivate(sim)
-// 			}
-// 		},
-// 	})
-
-// 	rageMetrics := druid.NewRageMetrics(actionID)
-
-// 	furorProcChance := []float64{0, 0.2, 0.4, 0.6, 0.8, 1}[druid.Talents.Furor]
-
-// 	druid.BearForm = druid.RegisterSpell(Any, core.SpellConfig{
-// 		ActionID: actionID,
-// 		Flags:    core.SpellFlagNoOnCastComplete | core.SpellFlagAPL,
-
-// 		ManaCost: core.ManaCostOptions{
-// 			BaseCost:   0.55,
-// 			Multiplier: (1 - 0.2*float64(druid.Talents.KingOfTheJungle)) * (1 - 0.1*float64(druid.Talents.NaturalShapeshifter)),
-// 		},
-// 		Cast: core.CastConfig{
-// 			DefaultCast: core.Cast{
-// 				GCD: core.GCDDefault,
-// 			},
-// 			IgnoreHaste: true,
-// 		},
-
-// 		ApplyEffects: func(sim *core.Simulation, _ *core.Unit, spell *core.Spell) {
-// 			rageDelta := 0 - druid.CurrentRage()
-// 			if sim.Proc(furorProcChance, "Furor") {
-// 				rageDelta += 10
-// 			}
-// 			if rageDelta > 0 {
-// 				druid.AddRage(sim, rageDelta, rageMetrics)
-// 			} else if rageDelta < 0 {
-// 				druid.SpendRage(sim, -rageDelta, rageMetrics)
-// 			}
-// 			druid.BearFormAura.Activate(sim)
-// 		},
-// 	})
-// }
+func (druid *Druid) registerBearFormSpell() {
+	actionID := core.ActionID{SpellID: core.TernaryInt32(druid.Level < 40, 5487, 9634)}
+
+	statBonus := druid.GetFormShiftStats().Add(stats.Stats{
+		stats.AttackPower: 3 * float64(druid.Level),
+		stats.Health:      core.TernaryFloat64(druid.Level < 40, (18*float64(druid.Level))-160, (32*float64(druid.Level))-680),
+	})
+
+	feralApDep := druid.NewDynamicStatDependency(stats.FeralAttackPower, stats.AttackPower, 1)
+
+	var hotwDep *stats.StatDependency
+	if druid.Talents.HeartOfTheWild > 0 {
+		hotwDep = druid.NewDynamicMultiplyStat(stats.Stamina, 1.0+0.04*float64(druid.Talents.HeartOfTheWild))
+	}
+
+	sotfdtm := 1.0
+	if druid.HasRune(proto.DruidRune_RuneChestSurvivalOfTheFittest) {
+		sotfdtm = 0.80
```

Comment:
So technically, [Survival of the Fittest](https://www.wowhead.com/classic/spell=411115/survival-of-the-fittest) works as two separate -10% damage modifiers. The first is always active, the second is only active in form.

So for bear, it's actually 0.9*0.9 = 0.81 damage mod from the rune.